verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -58,6 +58,31 @@ export class AuthController {
             },
         );
     }
+
+    // configureLocalAuth will get or create the user present in the request.
+    configureLocalAuth(): void {
+        console.log('Configuring local auth for tests');
+        // /register creates a user if necessary and log them in.
+        this.router.post(
+            '/register',
+            async (req: Request, res: Response): Promise<void> => {
+                const user = await User.create({
+                    name: req.body.name,
+                    email: req.body.email,
+                    // Necessary to pass mongoose validation.
+                    googleID: 42,
```

Comment:
Nice choice of user ID, the day after [towel day](http://www.towelday.org/) :)

Reply:
haha, didn't know there was such a day :) I'll wear a towel next year...